src/PhpSpec/Runner/Maintainer/CollaboratorsMaintainer.php

```diff
@@ -140,6 +130,9 @@ private function generateCollaborators(CollaboratorManager $collaborators, \Refl
                 if (($indexedClass = $this->getParameterTypeFromIndex($classRefl, $parameter))
                     || ($indexedClass = $this->getParameterTypeFromReflection($parameter))) {
                     $collaborator->beADoubleOf($indexedClass);
+                    if (null !== $indexedClass) {
```

Comment:
@bestform the value of an assignment is the value itself. And then, ``null`` is indeed a falsy value when casting it